packages/storage/src/ref.ts

```diff
@@ -91,19 +91,19 @@ class ReferenceStore implements AsyncReadable<RequestInit> {
 	}
 
 	static fromSpec(
-		spec: Record<string, unknown>,
+		spec: Promise<Record<string, unknown>>,
```

Comment:
I think similarly we could make this

```suggestion
		spec: Promise<Record<string, unknown>> | Record<string, unknown>,
```

it's fine to `await` a non Promise in the context of `get()` which is always async.

Reply:
This function could `Promise.resolve()` for consistency internally.